features/admin.tenants.v1/components/add-modal/add-tenant.tsx

```diff
@@ -38,6 +39,7 @@ import { AddTenantWizardForm, AddTenantWizardFormValuesInterface } from "../form
 interface AddTenantWizardPropsInterface extends TestableComponentInterface {
     openModal: boolean;
     onCloseHandler: () => void;
+    adus: ADU[];
```

Comment:
Let's not use abbreviations as interface members as it hurts readability. As previously suggested, shall we do something like this?

```suggestion
    deploymentUnits: DeploymentUnit[];
```